react-native-async-storage/async-storage
# ci(iOS): Fix yarn install failing with Xcode 12

Summary:
---------

`yarn install` fails because Detox fails to build with Xcode 12.

Resolves #454.

Test Plan:
----------

1. Make sure the repo is clean: `git clean -dfqx`
2. Run `yarn`
